version.json

```diff
@@ -0,0 +1,17 @@
+{
+  "vast-version-fallback_COMMENT": [
+    "The main version fragment. This should be identical to the closest",
+    "annotated git tag without the leading 'v'.",
+    "This value gets updated automatically by `scripts/prepare-release`.",
+    "If you modify it by hand you should also run the `update-changelog`",
+    "target and add the changes to CHANGELOG.md and the changelog directory",
+    "to the same commit."
+  ],
+  "vast-version-fallback": "2.3.0",
+  "vast-partition-version_COMMENT": [
+    "The partition version. This number must be bumped alongside the release",
+    "version for releases that contain major format changes to the on-disk",
+    "layout of VAST's partitions."
+  ],
```

Comment:
This is very ugly compared to what we had previously. Do you know if we can make this either YAML or JSON5 so we can actually use comments?

Reply:
:cry: I was actually quite proud of this solution when I came up with it. This file should really be JSON as that is much more widely supported than any of the alternatives.